src/osd/scrubber/pg_scrubber.cc

```diff
@@ -1524,10 +1548,15 @@ void PgScrubber::replica_scrub_op(OpRequestRef op)
 
   set_queued_or_active();
   advance_token();
+  const auto& conf = m_pg->get_cct()->_conf;
+  const int max_from_conf = size_from_conf(
+    m_is_deep, conf, "osd_scrub_chunk_max", "osd_shallow_scrub_chunk_max");
+  auto cost = get_scrub_cost(max_from_conf, m_is_deep);
   m_osds->queue_for_rep_scrub(m_pg,
 			      m_replica_request_priority,
 			      m_flags.priority,
-			      m_current_token);
+			      m_current_token,
+			      cost);
```

Comment:
OSDService::queue_for_rep_scrub names this last parameter "cost_per_object".  Here, however, cost appears to be the total cost of the chunk already multiplied by the number of objects.  Please ensure that the names are consistent.